content/docs/get-started/index.md

```diff
@@ -1,20 +1,24 @@
 ---
-description: 'Learn how you can use MLEM to easily manage and deploy models'
+description:
+  'Learn how to use MLEM to manage and deploy machine learning models easily.'
 ---
 
-# Get Started
+# Get Started with MLEM
 
-For this guide, we will need a Python environment with the following python
-packages installed: `pandas`, `scikit-learn`, `mlem[fastapi,heroku]`.
-Additionally, we'll require `docker` to be installed on your local machine to
-showcase building and deploying a containerized model server.
+<!--
+## Get Started with MLEM
+-->
+
+<admon type="info">
```

Comment:
This looks strange right at the top of the tutorial. Why do we need this admon?

Reply:
Well, it's 3rd-party tool documentation, so it should be separated somehow IMO. Would you prefer a hidden `<details>` section? I wouldn't hide it since the steps are required to follow the tutorial. Currently it looks like this:

<img width="1234" alt="image" src="https://user-images.githubusercontent.com/1477535/209610960-6ff551b1-e5dc-4cd6-911d-928cbe22aa56.png">

Reply:
> p.s. maybe if the admonition seems to loud to you we can use an old-school block quote (like this comment).